src/wwwroot/locales/en-translation.js

```diff
@@ -227,7 +227,7 @@
   "dkim_configuration_help_selector_title": "Your DKIM Selector:",
   "dkim_configuration_help_public_k_title": "Your DKIM Public Key:",
   "dkim_configuration_help_spf_title": "SPF",
-  "dkim_configuration_help_spf_text": "Add the following data to the ¨SPF¨ field: ¨include: fromdoppler.com¨",
+  "dkim_configuration_help_spf_text": "Add the following data to the ¨SPF¨ field: ¨include: spf.fromdoppler.com¨",
```

Comment:
Oh, there should not be a space between `include:` and `spf.spf.fromdoppler.com`